### PR TITLE
Unit test to Fused_embedding_fc_lstm op

### PR DESCRIPTION
**Pull request by jczaja**
This is followup PR to #13553.

It also introduce a fix to BatchCompute path of this operator

### CODE REVIEW

File: paddle/fluid/operators/fused_embedding_fc_lstm_op.cc [hunk 1]
```diff
@@ -93,11 +93,8 @@ void FusedEmbeddingFCLSTMOp::InferShape(
   ctx->SetOutputDim("Cell", out_dims);
   ctx->ShareLoD("Ids", "Hidden");
   ctx->ShareLoD("Ids", "Cell");
-  int xx_width;
-  if (ctx->Attrs().Get<bool>("use_seq")) {
-    xx_width = wh_dims[1];
-  } else {
-    xx_width = x_dims[1] > wh_dims[1] ? wh_dims[1] : x_dims[1];
+  int xx_width = wh_dims[1];
```
Comment (hshen14) [inline paddle/fluid/operators/fused_embedding_fc_lstm_op.cc]: Does it mean "x_dims[1] > wh_dims[1]" is always true？

Comment (tensor-tang) [reply paddle/fluid/operators/fused_embedding_fc_lstm_op.cc]: Why keep xx_width always be wh_dims[1]?

Comment (jczaja) [reply paddle/fluid/operators/fused_embedding_fc_lstm_op.cc]: @tensor-tang Good catch! It should be fixed now. Thanks!

@hshen14 This op is based on fusion_lstm_op.cc . And in this fusion lstm op there were two paths of  doing  Embedding*W_x operation depending on the dims of buffers. I think that was matter of optimization (I didn't really understand the reasons) . Anyway In this op: fused_embedding_fc_lstm We do not perform embedding_W_x , but just look it up in an array , so there is no need to evaluate condition that was removed. It is even worse, op was crashing when BatchCompute path was executed with this condition enabled.

**Comment by jczaja** [comment]
@luotao1 Could you please help with failing: MAC_Night build.

**Review by tpatejko — APPROVED** [review]
LGTM!

**Comment by luotao1** [comment]
@jczaja You can ignore the Mac Night build now. @tensor-tang Do you have any other suggestion?

**Review by tensor-tang — APPROVED** [review]

**Pull request merged**